consumers/notifiers/service_test.go

```diff
@@ -181,6 +193,8 @@ func TestListSubscriptions(t *testing.T) {
 				Subscriptions: subs[:3],
 				Total:         total,
 			},
+			identifyErr: nil,
+			identityRes: &magistrala.IdentityRes{Id: validID},
```

Comment:
User string instead of the full response; return response in the assert. Do the same for all the other test cases.